shopify_app.gemspec

```diff
@@ -10,25 +10,27 @@ Gem::Specification.new do |s|
   s.author      = "Shopify"
   s.summary     = "This gem is used to get quickly started with the Shopify API"
 
-  s.required_ruby_version = ">= 3.1"
+  s.required_ruby_version = ">= 3.2"
 
   s.metadata["allowed_push_host"] = "https://rubygems.org"
 
   s.add_runtime_dependency("addressable", "~> 2.7")
-  s.add_runtime_dependency("rails", "> 5.2.1")
+  s.add_runtime_dependency("rails", "> 7")
```

Comment:
Could you clarify what is the minimum version we would need to support to fix the issue?

In the changelog we specify rails 7.1

Reply:
Thanks for taking the initiative here! I can take it from here! 

Thank you for clarifying the intended version.

I tested this with our app template and there were no problems with it.